modules/effects/src/actions.ts

```diff
@@ -1,8 +1,11 @@
 import { Inject, Injectable } from '@angular/core';
 import { Action, ScannedActionsSubject } from '@ngrx/store';
-import { Observable, Operator, OperatorFunction } from 'rxjs';
+import { Observable, OperatorFunction, Operator } from 'rxjs';
 import { filter } from 'rxjs/operators';
 
+// Removes types from T that are not assignable to U.
+export type Filter<T, U> = T extends U ? T : never;
```

Comment:
Because we're using `Extract` this can be removed now.

Reply:
Indeed.